File: go/B4_13136.go
// https://www.acmicpc.net/problem/13136

package main

import (
	"bufio"
	"fmt"
	"math"
	"os"
)

func main() {
	rw := bufio.NewReadWriter(bufio.NewReader(os.Stdin), bufio.NewWriter(os.Stdout))
	defer rw.Flush()

	var r, c, n float64
	fmt.Fscan(rw, &r, &c, &n)

	fmt.Fprintf(rw, "%.0f", math.Ceil(r/n)*math.Ceil(c/n))
}

File: clang/B4_13136.c
// https://www.acmicpc.net/problem/13136

#include <stdio.h>
#include <math.h>

int main(void)
{
	float r, c, n;
	scanf("%f %f %f", &r, &c, &n);
	printf("%.0f", ceil(r / n) * ceil(c / n));
	return 0;
}
